### DISCUSSION TITLE
Conversation Etiquette?

### DISCUSSION THREAD

**Discussion by GaryKopp** [eclipse-emfcloud/emfcloud#94] [All other] [answered]
In my new home away from home, at least temporarily, I'd like to make sure I adequately acknowledge the answers I receive. Are simple "thank you's" encouraged, or is there another way I should express my appreciation? I don't want contributors to think I'm ignoring their help.

**Comment by planger** [comment, marked as answer]
Nice of you to ask! :) I think it would make a lot of sense to hit `Mark as answer` on a reply that answered your question.
For extra appreciation points, you could upvote and/or add an emoticon.
